packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -659,6 +660,15 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.deferJoin = undefined;
 
+    /**
+     * Staus of websocket connection/mercury connection.
+     * @instance
+     * @type {Boolean}
+     * @private
+     * @memberof Meeting
+     */
+    this.webSocketConnection = this.webex.internal.mercury.connected;
```

Comment:
Changes to mercury status will not be tracking in this variable. This should be a getter instead that returns the `mercury.connected` property.

Reply:
After reviewing the rest of the PR it looks like you want a way to track if the websocket has ever previously been connected. Perhaps a better name would be: 

```suggestion
    this.webSocketConnectionEstablished = this.webex.internal.mercury.connected;
```